ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -723,6 +723,13 @@ let migrate_send'  ~__context ~vm ~dest ~live ~vdi_map ~vif_map ~options =
          else acc)
       [] vm_and_snapshots in
 
+  (* Double check that all of the suspend VDIs are all visible on the source *)
```

Comment:
It's checking on the source, before the migration. Which makes me think that this check should be done in `assert_can_migrate`...?